src/io/fastx.rs

```diff
@@ -0,0 +1,442 @@
+use std::io;
+use std::io::prelude::*;
+use std::mem;
+
+use bio_types::sequence::SequenceRead;
+
+use crate::io::fasta;
+use crate::io::fasta::FastaRead;
+use crate::io::fastq;
+use crate::io::fastq::FastqRead;
+use crate::utils::TextSlice;
+
+const BASE_QUAL: u8 = 30;
```

Comment:
I had to find a value to use as a default quality (when going from FASTA -> FASTQ). What would be a good one?